app/models/concerns/income_summarisable.rb

```diff
@@ -39,6 +39,34 @@ def combined_disposable_income
     from_cfe_payload("result_summary.disposable_income.combined_total_disposable_income")
   end
 
+  def client_benefits
+    from_cfe_payload("assessment.gross_income.state_benefits.monthly_equivalents.all_sources.value")
+  end
+
+  def client_maintenance_in
+    from_cfe_payload("assessment.gross_income.other_income.monthly_equivalents.all_sources.maintenance_in.value")
+  end
+
+  def client_pensions
```

Comment:
When I run this locally, I get `Float does not have #dig method` error. 

So am assuming that this `dig` method can't be used on a float??


>   def from_cfe_payload(path)
>     session_data.dig("api_response", *path.split("."))
>   end
>

Reply:
fixed this, there was an issue with calling 'value'